### ISSUE TITLE
4.1.0 fails to handle comments at top of file

### ISSUE THREAD

**Issue by bendikro** [timothycrosley/isort#316]
isort version 4.1.0 fails to handle a file `__ini__.py` containing the following:
```python
"""Text"""
# This is a comment
import pkg_resources
```

```bash
isort --diff --recursive deluge/__init__.py
--- __init__.py:before	2015-08-31 00:39:53.843613
+++ __init__.py:after	2015-08-31 00:40:24.862446
@@ -1,3 +1,4 @@
 """Text"""
+
 # This is a comment
 import pkg_resources
```

When adding the requested empty line and rerunning:
```bash
isort --diff --recursive deluge/__init__.py
--- __init__.py:before	2015-08-31 00:44:11.900160
+++ __init__.py:after	2015-08-31 00:44:14.073559
@@ -1,4 +1,5 @@
 """Text"""
+

 # This is a comment
 import pkg_resources
```

**Comment by timothycrosley** [comment]
Thanks for reporting this issue, I'll make sure to have a fix in ASAP

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Fixed in release 4.1.1

Thanks again for identifying this issue!

~Timothy

**Comment by bendikro** [comment]
Thanks for the **quick** fix!
